File: app/src/main/java/com/jonasgerdes/stoppelmap/model/schedule/Events.java
package com.jonasgerdes.stoppelmap.model.schedule;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by Jonas on 04.08.2016.
 */
public class Events extends RealmObject {

    private RealmList<Event> events;

    public RealmList<Event> getEvents() {
        return events;
    }

    public void setEvents(RealmList<Event> events) {
        this.events = events;
    }
}

File: app/src/main/java/com/jonasgerdes/stoppelmap/model/events/Events.kt
package com.jonasgerdes.stoppelmap.model.events

import io.realm.RealmList
import io.realm.RealmObject



/**
 * @author Jonas Gerdes <dev@jonasgerdes.com>
 * @since 10.08.2017
 */
open class Events : RealmObject() {
    var events: RealmList<Event>? = RealmList()
}